datalad/interface/tests/test_add_archive_content.py

```diff
@@ -102,40 +105,51 @@ def test_add_archive_dirs(path_orig, url, repo_path):
     # change to repo_path
     with chpwd(repo_path):
         # create annex repo
-        repo = AnnexRepo(repo_path, create=True)
+        ds = Dataset(repo_path)
+        ds.create(force=True)
```

Comment:
Just FTR: Most tests do

```suggestion
        ds = Dataset(repo_path).create(force=True)
```

Not criticial, but more same more good.

Reply:
thx! I'll adjust this locally in one go